HandsOn/Group33/mappings/mapping.rml.ttl

```diff
@@ -0,0 +1,130 @@
+@base <http://www.semanticweb.org/RecargaMadrid/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix rr: <http://www.w3.org/ns/r2rml#> .
+@prefix rml: <http://semweb.mmlab.be/ns/rml#> .
+@prefix ql: <http://semweb.mmlab.be/ns/ql#> .
+@prefix transit: <http://vocab.org/transit/terms/> .
+@prefix wgs84_pos: <http://www.w3.org/2003/01/geo/wgs84_pos#> .
+@prefix classes: <http://www.semanticweb.org/RecargaMadrid/classes#> .
+@prefix properties: <http://www.semanticweb.org/RecargaMadrid/properties#> .
+
+<#TriplesMap1>
```

Comment:
faltan los object properties en el mapping (propiedades que relacionan una clase con otra, que en el mapping se definen como una relación entre un mapping con otro).
Podéis ver ejemplos aquí: https://rml.io/specs/rml/#logical-join

Faltan clases. Tenéis la clase Puesto en la ontología, donde está su TriplesMap? (y otras más)

Reply:
Ya está arreglado, muchas gracias :)